_episodes/01-numpy.md

```diff
@@ -136,7 +137,7 @@ print(weight_kg)
 ~~~
 {: .output}
 
-To change variable's value, we have to assign it a new one:
+To change value of the `weight_kg` variable, we have to assign it a new one:
```

Comment:
This could be "To change the value"...

Reply:
I'll disagree here. If we say `To change the value`, the complete sentence would read:
> To change the value, we have to assign it a new one

and `it` in the second half of the sentence becomes ambiguous.

Reply:
Ok, leaving it as you have it works for me.

Reply:
how about something like this:
> To change the value of the `weight_kg` variable, we have to **assign** `weight_kg` to a new value using the equals `=` sign.

Reply:
I think of assigning a value to a variable, not the other way around, so I suggest a slight change of removing "to":
"To change the value of the weight_kg variable, we have to assign weight_kg a new value using the equals = sign."